sendgrid/helpers/mail/mail.py

```diff
@@ -1,4 +1,5 @@
 """v3/mail/send response body builder"""
+from ..exceptions import SendGridException
```

Comment:
Actually I missed them after I finished my testing. I will push a commit which will correct this.